<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UEFITool</class>
 <widget class="QMainWindow" name="UEFITool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>UEFITool 0.10.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Structure</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QTreeView" name="structureTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="indentation">
          <number>10</number>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImageFile"/>
    <addaction name="actionSaveImageFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>A&amp;ction</string>
    </property>
    <widget class="QMenu" name="menuChangeCompressionTo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Change &amp;compression to</string>
     </property>
     <addaction name="actionChangeToNone"/>
     <addaction name="actionChangeToTiano"/>
     <addaction name="actionChangeToEfi11"/>
     <addaction name="actionChangeToLzma"/>
    </widget>
    <addaction name="actionRebuild"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionExtract"/>
    <addaction name="actionExtractBody"/>
    <addaction name="actionExtractUncompressed"/>
    <addaction name="separator"/>
    <addaction name="actionInsertInto"/>
    <addaction name="actionInsertAfter"/>
    <addaction name="actionInsertBefore"/>
    <addaction name="separator"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="menuChangeCompressionTo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>H&amp;elp</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="infoDockWidget">
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="infoEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="centerOnScroll">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="messageDockWidget">
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>113</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>113</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item>
      <widget class="QListWidget" name="messageListWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionInsertAfter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert &amp;after...</string>
   </property>
   <property name="toolTip">
    <string>Insert an object from file after selected object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionInsertBefore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert &amp;before...</string>
   </property>
   <property name="toolTip">
    <string>Insert object from file before selected object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rep&amp;lace</string>
   </property>
   <property name="toolTip">
    <string>Replace selected object with an object from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>E&amp;xtract as is...</string>
   </property>
   <property name="toolTip">
    <string>Extract selected object as is to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExtractBody">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Extract &amp;without header...</string>
   </property>
   <property name="toolTip">
    <string>Extract selected object without header to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionExtractUncompressed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Extract &amp;uncompressed...</string>
   </property>
   <property name="toolTip">
    <string>Extract selected FFS file uncompressing it </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Re&amp;move</string>
   </property>
   <property name="toolTip">
    <string>Remove selected object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionOpenImageFile">
   <property name="text">
    <string>&amp;Open image file...</string>
   </property>
   <property name="toolTip">
    <string>Open image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionInsertInto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert &amp;into...</string>
   </property>
   <property name="toolTip">
    <string>Insert object from file into selected object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSaveImageFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save image file...</string>
   </property>
   <property name="toolTip">
    <string>Save modified image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRebuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Rebuild</string>
   </property>
   <property name="toolTip">
    <string>Rebuild selected object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionChangeToTiano">
   <property name="text">
    <string>&amp;Tiano</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionChangeToEfi11">
   <property name="text">
    <string>&amp;EFI 1.1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionChangeToLzma">
   <property name="text">
    <string>&amp;LZMA</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About UEFITool</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionChangeToNone">
   <property name="text">
    <string>&amp;Uncompressed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
